langwatch/src/server/elasticsearch/transformers.ts

```diff
@@ -105,19 +124,37 @@ export const transformElasticSearchTraceToTrace = (
   };
 };
 
-export const transformElasticSearchSpanToSpan = (esSpan: ElasticSearchSpan, protections: Protections): Span => {
+export const transformElasticSearchSpanToSpan = (
+  esSpan: ElasticSearchSpan,
+  protections: Protections,
+  redactions: string[]
+): Span => {
   const { input, output, metrics, ...spanFields } = esSpan;
 
   let transformedInput: SpanInputOutput | null = null;
   let transformedOutput: SpanInputOutput | null = null;
   let transformedMetrics: SpanMetrics | null = null;
 
-  if (input && protections.canSeeCapturedInput === true) {
-    transformedInput = elasticSearchToTypedValue(input);
+  if (input) {
+    transformedInput =
```

Comment:
this is redundant as this is done in the query, imo

Reply:
not really, we were setting it to null, this was caused the input from completely disapear from the frontend, now by setting it instead to "[REDACTED]", it shows nicely on the frontend with the little lock icon you did

Reply:
Is that the case? It returns null as I wanted to differentiate for the potential case that the actual message was `[REDACTED]`. Super niche, I know.

The `RedactedField` component shows based on the setting on the project, it doesn't look at the data at all. 

I think i'm just misunderstanding this change.

Reply:
the thing is there is a span.input check even before it reaches `<RedactedField>`, as we want to hide the field entirely if there was really no input
![image](https://github.com/user-attachments/assets/2bb04490-2d96-4200-a9ec-fd3b1c372199)

I think the case where input is `null` on purpose is WAY more common than the case where it is `[REDACTED]`, however, for this case, it will show as expected anyway, it's really not a conflict